commands/host.vegman

```diff
@@ -224,6 +224,84 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+
+function loc_config_default_set {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Host BIOS settings will be reset to default on next boot."
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+
+  ipmitool chassis bootdev none clear-cmos=yes >/dev/null
+}
+
+function loc_config_default_cancel {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Cancel reset to default host settings on next boot."
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+
+  ipmitool chassis bootdev none >/dev/null
+}
+
+function loc_config_default_status {
+  expect_noarg "$@"
+  strArr=($(ipmitool chassis bootparam get 5|grep -i 'Boot parameter data'))
+  val=${strArr[3]}
+  len=${#val}
+
+  if [[ $((16#${val:$len-8:1})) -ge 8 ]]; then
+    echo "BIOS settings will be RESET to defaults on next boot"
+  else
+    echo "BIOS settings will NOT be reset to defaults"
+  fi
+}
+
+# @sudo cmd_config admin,operator,user
+# @doc cmd_config
+# Host configuration
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_default admin,operator,user
+# @doc cmd_config_default
+# Host set/reset UEFI boot configuration on next boot.
+# Possible flags: --cancel, --status
```

Comment:
I assume the options should be explained more detailed.
And `--yes` also should present here.
This comment is used as a help message shown by `host config default --help`.